commons/src/methods/isNonInlineAttachment.ts

```diff
@@ -0,0 +1,14 @@
+import type { Message, File } from "@commons/types/Nylas";
+import {
+  DisallowedContentTypes,
+  InlineImageTypes,
+} from "@commons/constants/attachment-content-types";
+
+export const isNonInlineAttachment = (message: Message, file: File): boolean =>
```

Comment:
Only thing is this function name - `isNon` is strange. But I can't seem to think of a better name :( 

Would be ideal if we could detect `isInlineAttachment` and then just use the negative.

Reply:
@AaronDDM I think this can be more succinctly named `isAttachedFile`. WDYT?

Reply:
Yes!